### PR TITLE
Add helper text and sampleConfig.json file

### LINKED ISSUE #285: Add explanation and sample JSON file for "Advanced Config"
Badgeyay has an Advanced field "Upload a config file (JSON) to customise badges". Please provide some helper text about what can be configured additionally through the JSON file and add a link to a sample JSON file.

### PR DESCRIPTION
**Pull request by mohdomama**
Fixes #285

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
- Add helper text. This is a mouseover text that appears when user hovers on ` Upload a config file (JSON) to customise badges `
- Add sampleConfig.json file in the sample folder and provide its link below ` Upload a config file (JSON) to customise badges `
- The following is the screenshot of the changes :
new

### CODE REVIEW

**Comment by gabru-md** [comment]
@mohdomama can you add a bit of css to the tooltip to make it look more elegant.
Like white background with black text and some padding would look great.

**Comment by mohdomama** [comment]
@gabru-md you sure white background will look good ? Since our websites background is also white, it might not look distinctive.

**Comment by gabru-md** [comment]
Adding border and shadow to the box. 
I'm just proposing these changes. You can wait till other maintainers comment.
Btw thanks for the PR

**Comment by mohdomama** [comment]
@gabru-md  I will work on it. Thanks for the review :)

**Comment by mohdomama** [comment]
@vaibhavsingh97 I have used popover. I have attached an image of it. The problem is when i put the CSS in the external style.css file it doesn't work. It only works when i the css is in the layout.html itself. 
pop2

**Comment by vaibhavsingh97** [comment]
@mohdomama check [this](goo.gl/MjZiwK)

**Comment by djmgit** [comment]
@mohdomama please check the above reported issue

File: app/static/css/style.css [hunk 1]
```diff
@@ -1,3 +1,76 @@
+body, span, div, table, tr, tbody, tfoot, td, hr { 
+    background-color:  exact;
+    color: exact;
```
Comment (djmgit) [inline app/static/css/style.css]: Please maintain a consistent indentation.

File: app/static/css/style.css [hunk 2]
```diff
@@ -1,3 +1,76 @@
...
+  display: none;
+  max-width: none;
+  width: wrap;
+  padding: 1px;
+  text-align: left;
+  white-space: normal;
+  background-color: #FFF;
+  color: #000000; 
+  -webkit-background-clip: padding-box;
+          background-clip: padding-box;
```
Comment (djmgit) [inline app/static/css/style.css]: Please indent this line

File: app/static/css/style.css [hunk 3]
```diff
@@ -1,3 +1,76 @@
...
+}
+.popover1-content {
+  background-color:  #FFF;
+  color: #000000;
+  padding: 18px;
+  border-radius: 0 0 5px 5px;
+  border: 2px solid #D3D3D3
+}
+.popover1 > .arrow {
+border-width: 11px;
```
Comment (djmgit) [inline app/static/css/style.css]: Indentation problem.

File: app/static/css/style.css [hunk 4]
```diff
@@ -1,3 +1,76 @@
...
+}
+.popover1 > .arrow:after {
+content: "";
+border-width: 10px;
```
Comment (djmgit) [inline app/static/css/style.css]: Indentation problem

**Comment by mohdomama** [comment]
@djmgit Working on it :)

**Comment by mohdomama** [comment]
Houndci is reporting some issues in code that i didnt write. Cant do anything about it.

**Comment by vaibhavsingh97** [comment]
@mohdomama Still can you please fix those issues

**Comment by jajodiaraghav** [comment]
@mohdomama Can you please fix the conflicts so that this can be merged

**Comment by ParthS007** [comment]
closing it with reference to #346

**Comment by mohdomama** [comment]
Sorry folks, I was off the grid for some time. It appears @ParthS007 has solved it. All credits to him :+1:

**Pull request closed without merging**